downstream/assemblies/eda/assembly-eda-logging-strategy.adoc

```diff
@@ -0,0 +1,11 @@
+[id="eda-logging-strategy"]
+
+= {EDAName} logging strategy
+
+{EDAName} offers an audit logging solution over its resources. 
+We log each supported create, read, update, and delete (CRUD) operation done against rulebook activations, event streams, decision environments, projects, and activations. 
```

Comment:
Suggest changing sentence to remove "We". Something like,

"Each supported create, read, update and delete (CRUD) operation is logged against rulebook activations, ..."